js/apps/system/_admin/aardvark/APP/react/src/views/analyzers/addAnalyzer/AddAnalyzerForm.tsx

```diff
@@ -17,12 +17,16 @@ import { useAnalyzersContext } from "../AnalyzersContext";
 import { TYPE_TO_LABEL_MAP } from "../AnalyzersHelpers";
 import { AnalyzerTypeForm } from "./AnalyzerTypeForm";
 
-const ANALYZER_TYPE_OPTIONS = Object.keys(TYPE_TO_LABEL_MAP).map(type => {
-  return {
-    value: type,
-    label: TYPE_TO_LABEL_MAP[type as AnalyzerTypes]
-  };
-});
+const ANALYZER_TYPE_OPTIONS = Object.keys(TYPE_TO_LABEL_MAP)
+  .map(type => {
+    const excludedTypes = ["minhash"];
+    if (excludedTypes.includes(type)) return null;
```

Comment:
excluding for now, will add support in the next PR, there's also a separate task for the same: [FE-314](https://arangodb.atlassian.net/browse/FE-314)